### ISSUE TITLE
Browser console errors when using Oban Web UI

### ISSUE THREAD

**Issue by marceldegraaf** [sorentwo/oban#1130]
### Environment

- Oban Version:
  - `oban 2.17.12`
  - `oban_pro 1.4.10`
  - `oban_web 2.10.5`
- PostgreSQL Version
  - PostgreSQL 15.7 (using Google Cloud SQL)
- Elixir & Erlang/OTP Versions (`elixir --version`)
  - Erlang/OTP 27
  - Elixir 1.17.2
- Browser
  - Firefox 128.0.3 on macOS Sonoma 14.5

### Current Behavior

We're hosting the Oban Web dashboard in our app's router, setting the socket path to match our LiveView configuration:

```
  scope "/oban", as: :oban do
    pipe_through([:browser, :require_user])

    oban_dashboard("/", socket_path: "/socket/live")
  end
```

This generally works: we can load the Oban dashboard, open the various "States" pages (e.g. executing, available, scheduled), and inspect queues on the "Queues" tab. 

However, some actions do nothing in the UI, and upon inspection reveal a Javascript error in the browser console:

```
no component found matching phx-target of X
```

(where `X` is an integer)

We see this error when opening a retryable job and clicking the "Retry" button, when attempting to click the "Retryable Jobs" checkbox to select all retryable jobs, or when clicking the up/down arrows to configure the Local limit for a queue.

### Expected Behavior

We expect the actions in the UI to work, and not trigger an error in the browser's console.

**Comment by sorentwo** [comment]
@marceldegraaf Which version of Phoenix Live View are you using? IIRC, that error indicates a mismatch between LV versions.

**Comment by marceldegraaf** [comment]
We're using LiveView `0.20.1`. Looks like that's an older version, and Oban uses the latest 0.20 of LiveView (`~> 0.20`).

Upgrading LiveView locally seems to resolve the issue; I'll fix the deprecation warnings related to the LiveView upgrade, push to production, and report back if that indeed solves our issues.

**Comment by sorentwo** [comment]
I believe there are changes in LV v0.20.3 specifically that fix that issue, but it may be v0.20.2. Soon we'll be able to specify a proper v1.0+ version!

Closing this issue for now. Please report back if there are other problems.

**Comment by marceldegraaf** [comment]
Upgrading our application to the latest LiveView, matching the version Oban Web uses, has indeed fixed the issue on production. Thanks!
